data/lavezzo2020suppression/lavezzo2020suppression-extraction.md

```diff
@@ -0,0 +1,238 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python
+import yaml
+import pandas as pd
+from shedding_hub import folded_str
+import numpy as np
```

Comment:
Are all numpy, textwrap, and datetime used?

Reply:
Yes